Zappy_IA_Report/NN/Code/Tutorial4.py

```
@@ -0,0 +1,31 @@
+	return " ".join([reverse_word_index.get(i, "?") for i in text])
+
+train_data = keras.preprocessing.sequence.pad_sequences(train_data, value=word_index["<PAD>"], padding="post", maxlen=250)
+test_data = keras.preprocessing.sequence.pad_sequences(test_data, value=word_index["<PAD>"], padding="post", maxlen=250)
+
+model = keras.Sequential()
+model.add(keras.layers.Embedding(88000, 16))
+model.add(keras.layers.GlobalAveragePooling1D())
+model.add(keras.layers.Dense(16, activation="relu"))
+model.add(keras.layers.Dense(1, activation="sigmoid"))
+
+model.summary()
```

Flake8 (7.2.0)

[warning] no newline at end of file

(W292)